src/v/pandaproxy/error.cc

```diff
@@ -206,10 +199,3 @@ const std::error_category& reply_category() noexcept {
 }
 
 } // namespace pandaproxy
-
-namespace kafka {
-std::error_code make_error_code(kafka::error_code ec) {
-    return {static_cast<int>(ec), pandaproxy::kafka_error_category};
-}
-
-} // namespace kafka
```

Comment:
nit, might need a new line here.

Reply:
The diff adds a newline that previously wasn't there.

Reply:
no, i meant, we need to add a new line at end of this file, github warns if it fails to see a new line at end of source file like
<img width="263" alt="image" src="https://user-images.githubusercontent.com/59071/168186107-f52f5d1d-7b2c-45e6-8266-87cf0d12784a.png">

Reply:
I don't know how to explain it more. I agree with you. You're commenting on the existing file. The updated file has a newline.

Reply:
my apologies. i am very sorry. was looking at the wrong column.